### ISSUE TITLE
Installation on Ubuntu 20.04 fails

### ISSUE THREAD

**Issue by JakobStr** [UltraStar-Deluxe/USDX#553]
When trying to run `sudo apt-get update && sudo apt-get install git automake make gcc fpc libsdl2-image-dev libavformat-dev libswscale-dev libsqlite3-dev libfreetype6-dev portaudio19-dev libportmidi-dev liblua5.3-dev libopencv-videoio-dev`
the following errors occur:

`Reading package lists... Done
Building dependency tree       
Reading state information... Done
automake is already the newest version (1:1.16.1-4ubuntu6).
gcc is already the newest version (4:9.3.0-1ubuntu2).
gcc set to manually installed.
git is already the newest version (1:2.25.1-1ubuntu3).
make is already the newest version (4.2.1-1.2).
fpc is already the newest version (3.0.4+dfsg-23).
fpc set to manually installed.
portaudio19-dev is already the newest version (19.6.0-1build1).
libfreetype6-dev is already the newest version (2.10.1-2ubuntu0.1).
libsqlite3-dev is already the newest version (3.31.1-4ubuntu0.2).
libavformat-dev is already the newest version (7:4.2.4-1ubuntu0.1).
libswscale-dev is already the newest version (7:4.2.4-1ubuntu0.1).
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libopencv-videoio-dev : Depends: libopencv-imgcodecs-dev (= 4.2.0+dfsg-5) but it is not going to be installed
                         Depends: libopencv-videoio4.2 (= 4.2.0+dfsg-5) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.`

This happens on both of my Linux devices. 
`./autogen.sh` works without problems, but  
`./configure` fails as expected:
checking whether make sets $(MAKE)... yes
checking whether ln -s works... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for a BSD-compatible install... /usr/bin/install -c
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for cmake... /usr/bin/cmake
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for fpc... fpc
checking for fpcmake... fpcmake
checking version of fpc... [3.0.4]
checking whether the Free Pascal Compiler works... yes
checking whether the Free Pascal Compiler can link... yes
checking whether FPC supports -k"-z noexecstack"... yes
checking whether FPC supports -k"--copy-dt-needed-entries"... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for ranlib... ranlib
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for sdl2... no
configure: error: 

Package sdl2 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sdl2.pc'
to the PKG_CONFIG_PATH environment variable
No package 'sdl2' found

Alternatively, you may set --with-sdl2=nocheck and the environment
variables sdl2_[...] (see configure --help)
to appropriate values to avoid the need to call pkg-config.

See the pkg-config man page for more details.

Does somebody know how to fix this? Or get to run Ultrastar on Linux differently. 
In Wine the paths do not work, so this is not a good workaround either. 

Provide some additional information:
Any help appreciated.

- Operating System + version: Ubuntu 20.04

**Comment by s09bQ5** [comment]
The `apt-get install` didn't finish because for some reason it didn't want to install the dependencies libopencv-imgcodecs-dev and libopencv-videoio4.2. If you fix that and retry the apt-get install, it should install libsd2-image-dev, which puills in libsdl2-dev. When libsdl2-dev is installed, configure will no longer complain that sdl2 is missing.

**Comment by JakobStr** [comment]
Thank for your fast help @s09bQ5 . Yes I tried to install these packages, but the `The following packages have unmet dependencies: ..` error proceeded as I installed the required packages. In the end aptitude instead of apt-get solved the problem instantly. 

Got the solution in this thread: 
https://askubuntu.com/questions/363200/e-unable-to-correct-problems-you-have-held-broken-packages

For anyone who faces the same problem.
